docs/modules/community/pages/mailing-lists.adoc

```diff
@@ -0,0 +1,165 @@
+debate that should not be brought up on the mailing lists. You can examine opposing points of view condemning our
+convention and condoning it. Bringing this up for debate on a mailing list will add nothing new and is considered off-topic.
+
+*Do not cross post messages*
+
+In other words, pick one mailing list and send your messages to that mailing list only. Do not send your messages to
+multiple mailing lists. The reason is that people may be subscribed to one list and not to the other. Therefore, some
+people will only see part of the conversation.
+
+== James mailing lists
+
+=== Server user list
+
+*Low traffic*
+mailto:server-user-subscribe@james.apache.org[Subscribe]
+mailto:server-user-unsubscribe@james.apache.org[Unsubscribe]
+http://www.mail-archive.com/server-user@james.apache.org[Archive]
+
+This is the list where users of the Apache James (Server) meet and discuss issues. Developers are also expected to be
+subscribed to this list to offer support to users of Apache James (Server).
+
+=== Server developer list
+
+*High traffic*
+mailto:server-dev-subscribe@james.apache.org[Subscribe]
+mailto:server-dev-unsubscribe@james.apache.org[Unsubscribe]
+http://www.mail-archive.com/server-dev@james.apache.org[Archive]
+
+This is the list where participating developers of the Apache James Project meet and discuss issues, code changes/additions,
+etc. Subscribers to this list get notices of each and every code change, build results, testing notices, etc. Do not
+send mail to this list with usage questions or configuration problems -- that's what server-user@james is for.
+
+=== Mime4j
+
+*Medium traffic*
+mailto:mime4j-dev-subscribe@james.apache.org[Subscribe]
+mailto:mime4j-dev-unsubscribe@james.apache.org[Unsubscribe]
+http://mail-archives.apache.org/mod_mbox/james-mime4j-dev[Archive]
+
+Discussions on the http://james.apache.org/mime4j[Mime4j] parser library.
+
+=== Site developer list
+
+*Low traffic*
+mailto:site-dev-subscribe@james.apache.org[Subscribe]
+mailto:site-dev-unsubscribe@james.apache.org[Unsubscribe]
+http://www.mail-archive.com/site-dev@james.apache.org[Archive]
+
+This is the list where participating developers of the Apache James project discuss changes/additions to the James
+websites etc. Subscribers to this list get notifications of every commit of the website resources. Do not send mail to
+this list with James software problems -- that's what server-user@james is for.
+
+=== James general list
+
+*Low traffic*
+mailto:general-subscribe@james.apache.org[Subscribe]
+mailto:general-unsubscribe@james.apache.org[Unsubscribe]
+http://www.mail-archive.com/general@james.apache.org[Archive]
+
+This is the list for general discussions related to the running of the project, it is the public list of the
+James project management committee (PMC) and is a public list opened to all. Do not send mail to this list with James
+software problems -- that's what server-user@james is for.
+
+=== Mailet API list
```

Comment:
Should we keep that mailing list? It is not active.... Maybe it is better to discuss mailet pi concerns in server-dev....

Reply:
Remove! 😀

Reply:
:-)

though removal might need ML discussions ^^